roles/install_dbserver/tasks/install_dbserver.yml

```diff
@@ -1,13 +1,34 @@
 ---
 
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('Red Hat Enterprise Linux', 'RHEL') }}{{ ansible_distribution_major_version }}"
+
+- name: Check support for Operating System
+  fail:
+    msg: "Operating System = {{ os }} not supported."
+  when: os not in supported_os
+
+- name: Check supported versions for Database engine
+  fail:
+    msg: "Database Engine Version = {{ pg_version }} not supported.
+          Supported versions are {{ supported_pg_version }}"
+  when: pg_version|int not in supported_pg_version
+
+- name: Check if Database Engine is supported
+  fail:
+    msg: "Database Engine = {{ pg_type }} {{ pg_version }}
+          is not supported on RHEL8/CentOS8."
+  when:
+    - pg_type == 'EPAS' and pg_version|int < 12
+    - os in ['RHEL8','CentOS8']
+
 - name: Set force_install based on the user input
```

Comment:
Let's cleanup this task and respective files for our cleanup role

Reply:
Same as https://github.com/EnterpriseDB/edb-ansible/pull/85#discussion_r546015786